build/debian-iptables/Dockerfile

```diff
@@ -19,8 +19,8 @@ FROM BASEIMAGE
 CROSS_BUILD_COPY qemu-ARCH-static /usr/bin/
 
 RUN clean-install \
-    iptables \
-    ebtables \
     conntrack \
-    module-init-tools \
-    ipset
+    ebtables \
+    ipset \
+    iptables \
```

Comment:
can we add ipvsadm to the list?  Will make life easier when we switch to ipvs

Reply:
sure. that will want another tag bump, of course. 🙃

Reply:
discussed offline, will punt for now. I don't want to conflate bug fix and feature request during code slush.